src/Pages/AllRecipes/index.tsx

```diff
@@ -0,0 +1,9 @@
+import Header from "../../Components/Header";
+
+export default function AllRecipes() {
+  return (
+    <>
+      <Header title="All Recipes" />
```

Comment:
The double header thing still looks bad.

Reply:
Done